httpobs/database/database.py

```diff
@@ -140,15 +140,17 @@ def insert_test_results(site_id: int, scan_id: int, tests: list, response_header
                         (site_id, scan_id, name, expectation, result, passed, dumps(test), score_modifier))
 
         # Now we need to update the scans table
-        score, grade = get_grade_for_score(score)
+        score, grade, likelihood_indicator = get_grade_and_likelihood_indicator_for_score(score)
```

Comment:
Sorry I thought I had asked for this earlier -- can we use get_grade_and_likelihood, to keep the function name reasonable?  Thanks!